app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java

```diff
@@ -37,14 +37,17 @@
 import org.gdg.frisbee.android.adapter.DrawerAdapter;
 import org.gdg.frisbee.android.app.App;
 import org.gdg.frisbee.android.app.OrganizerChecker;
-import org.gdg.frisbee.android.special.SpecialEventActivity;
-import org.gdg.frisbee.android.special.TaggedEvent;
+import org.gdg.frisbee.android.eventseries.TaggedEventSeries;
+import org.gdg.frisbee.android.eventseries.TaggedEventSeriesActivity;
+
+import java.util.ArrayList;
 
 import butterknife.InjectView;
+import butterknife.OnItemClick;
 import de.keyboardsurfer.android.widget.crouton.Crouton;
 import de.keyboardsurfer.android.widget.crouton.Style;
 
-public abstract class GdgNavDrawerActivity extends GdgActivity {
+public abstract class GdgNavDrawerActivity extends GdgActivity{
```

Comment:
Good catch. I was working on checkstyle integration. It is almost finished but I will finish it after Release 2.0